pydantic/typing.py

```diff
@@ -67,6 +67,13 @@ def evaluate_forwardref(type_: ForwardRef, globalns: Any, localns: Any) -> Any:
         # TypeError: ForwardRef._evaluate() missing 1 required keyword-only argument: 'recursive_guard'
         return cast(Any, type_)._evaluate(globalns, localns, recursive_guard=set())
 
+else:
+
+    def evaluate_forwardref(type_: ForwardRef, globalns: Any, localns: Any) -> Any:
+        # Pydantic 1.x will not support PEP 695 syntax, but provide `type_params` to avoid
+        # warnings:
```

Comment:
```suggestion
        # Pydantic 1.x will not support PEP 695 syntax, but provide `type_params` to avoid warnings
```

Thanks for the comment, looks good